imap_processing/ultra/l1b/ultra_l1b_extended.py

```diff
@@ -244,9 +244,10 @@ def get_ph_tof_and_back_positions(
         stop_type_bottom
     ] + get_image_params("TOFBTOFF")  # 10*ns
 
+    # Variable xf_ph divided by 10 to convert to mm.
```

Comment:
can you add this comment to above line change as well? I was wondering that and then saw this comment.